plugin/projectionist.vim

```diff
@@ -161,7 +161,7 @@ augroup projectionist
         \   call s:Detect(b:NERDTree.root.path.str()) |
         \ endif
   autocmd VimEnter *
-        \ if get(g:, 'projectionist_vim_enter', 1) && argc() == 0 |
+        \ if get(g:, 'projectionist_vim_enter', 1) && argc() == 0 && v:this_session == '' |
```

Comment:
Stylistic nit: could you make this `empty(v:this_session)`?

Reply:
Just updated.